javaHapiValidatorLambda/serverless.yml

```diff
@@ -23,16 +23,29 @@ provider:
   environment:
     FHIR_VERSION:
       !Ref FhirVersion
-
+    IMPLEMENTATION_GUIDES_BUCKET: !ImportValue ImplementationGuidesBucketNameKey
+  iam:
+    role:
+      statements:
+        - Effect: "Allow"
+          Action:
+            - "s3:PutObject"
+            - "s3:GetObject"
+            - "s3:CreateBucket"
+            - "s3:ListBucket"
+            - "s3:ListObjects"
+          Resource: 
+            - !Join ['', ['arn:aws:s3:::', !ImportValue ImplementationGuidesBucketNameKey, '/*']]
+            - !Join ['', ['arn:aws:s3:::', !ImportValue ImplementationGuidesBucketNameKey]]
 
 package:
   artifact: target/fwoa-hapi-validator-dev.jar
 
 functions:
   validator:
     handler: software.amazon.fwoa.Handler
-    timeout: 25
-    memorySize: 2048
+    timeout: 300
+    memorySize: 5120
```

Comment:
Add documentation in the implementation guide doc suggesting updating the memory size here if user are facing IG size issue?